app/models/hearing.rb

```diff
@@ -3,10 +3,14 @@
 class Hearing < ApplicationRecord
   include BuilderMappable
   include CourtCentreRelatable
+
+  JURISDICTION_TYPES = %w[MAGISTRATES CROWN].freeze
+  LANGUAGES = %w[ENGLISH WELSH].freeze
+
   belongs_to :hearing_type
   belongs_to :cracked_ineffective_trial, optional: true
   has_many :prosecution_case_hearings, dependent: :destroy
-  has_many :prosecution_cases, through: :prosecution_case_hearings
+  has_many :prosecution_cases, through: :prosecution_case_hearings, inverse_of: :hearings
```

Comment:
can a hearing have more than one case?

Reply:
According to the CP structure, yes, however, we haven't encountered anything of that sort in the wild.